merge_test.go

```diff
@@ -196,7 +196,7 @@ func testMergeCheck(t *testing.T, merged, data1, data2 interface{}) {
 			if data2Map != nil {
 				data2Item = data2Map[name]
 			}
-			t.Logf("%v : %v : %v : %v", name, mergedItem, data1Item, data2Item)
+			//t.Logf("%v : %v : %v : %v", name, mergedItem, data1Item, data2Item)
```

Comment:
Intentionally commented?

Reply:
yeah, to remove excessive output during tests